assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
 che.infra.kubernetes.cluster_domain=NULL
 
-# Defines the way how servers are exposed to the world in Kubernetes infra.
-# List of strategies implemented in {prod-short}: `default-host`, `multi-host`, `single-host`.
+# Defines how servers are exposed to the world in Kubernetes infrastructure.
+# List of strategies implemented in {prod-short}: `default-host`, `multi-host`, and `single-host`.
 che.infra.kubernetes.server_strategy=multi-host
 
-# Defines the way in which the workspace plugins and editors are exposed in the single-host mode.
+# Defines the way in which the workspace plug-ins and editors are exposed in the single-host mode.
 # Supported exposures:
 # `native`:: Exposes servers using Kubernetes Ingresses. Works only on Kubernetes.
 # `gateway`:: Exposes servers using reverse-proxy gateway.
 che.infra.kubernetes.singlehost.workspace.exposure=native
 
-# Defines the way how to expose devfile endpoints, as end-user's applications, in single-host server strategy.
+# Defines the way to expose devfile endpoints as end-user's applications, in single-host server strategy.
 # They can either follow the single-host strategy and be exposed on subpaths, or they can be exposed on subdomains.
-# `multi-host`:: expose on subdomains
-# `single-host`:: expose on subpaths
+# Supported exposures:
+# `multi-host`:: Exposes devfile endpoints on subdomains.
+# `single-host`:: Exposes devfile endpoints on subpaths.
 che.infra.kubernetes.singlehost.workspace.devfile_endpoint_exposure=multi-host
 
-# Defines labels which will be set to ConfigMaps configuring single-host gateway.
+# Defines labels which are set to ConfigMaps configuring single-host gateway.
 che.infra.kubernetes.singlehost.gateway.configmap_labels=app=che,component=che-gateway-config
 
-# Used to generate domain for a server in a workspace in case property `che.infra.kubernetes.server_strategy` is set to `multi-host`
+# Generates a domain for a server in a workspace in case the property `che.infra.kubernetes.server_strategy` is set to `multi-host`
 che.infra.kubernetes.ingress.domain=
 
 # Indicates whether {prod-short} server is allowed to create {orch-namespace} for user
 # workspaces, or they're intended to be created manually by cluster administrator.
-# This property is also used by the OpenShift infra.
+# This property is also used by OpenShift infrastructure.
 che.infra.kubernetes.namespace.creation_allowed=true
 
 # Defines Kubernetes default namespace in which user's workspaces are created
 # if user does not override it.
-# It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`).
-# In that case, new namespace will be created for each user.
-# Used by OpenShift infra as well to specify a Project.
+# It's possible to use `<username>` and `<userid>` placeholders, for example: `che-workspace-<username>`.
+# In that case, new namespace is created for each user.
```

Comment:
This will ensure that a new namespace is created for each user, so that their workspaces do not collide with other users.